### ISSUE TITLE
Go to Symbol in Editor... - Ctrl+F12 keybinding is not working anymore in markdown files

### ISSUE THREAD

**Issue by agostonbarna** [kasecato/vscode-intellij-idea-keybindings#211]
The Ctrl+F12 keybinding used to trigger the *Go to Symbol in Editor..* command for markdown headers
![image](https://user-images.githubusercontent.com/5561396/110102486-23e1ab80-7da5-11eb-90e1-6c643ea93d42.png) but not anymore.

As a workaround I removed the `editorHasDefinitionProvider` condition from the `when` criteria of the `workbench.action.gotoSymbol` keybinding:
```json
{
  "key": "ctrl+f12",
  "command": "workbench.action.gotoSymbol",
  "when": "editorTextFocus"
},
{
  "key": "ctrl+f12",
  "command": "-workbench.action.gotoSymbol",
  "when": "editorHasDefinitionProvider && editorTextFocus"
}
```
But it would be nice if this would be the default in the extension.

Note that the default *Go to Symbol in Editor...* keybinding doesn't have a `when` condition at all:
![image](https://user-images.githubusercontent.com/5561396/110103049-c437d000-7da5-11eb-9be2-069acba3eaf2.png)

---

IntelliJ IDEA Keybindings version: v1.3.0
VS Code version: 1.53.2
OS: Linux

**Comment by agostonbarna** [comment]
Wow, thanks for the quick fix @kasecato :+1:

**Comment by kasecato** [comment]
@agostonbarna Just released v1.4.0. Please check it

**Comment by agostonbarna** [comment]
@kasecato: thanks, after upgrading to v1.4.0 I can confirm that the Ctrl+F12 keybinding is now working fine without my workaround :1st_place_medal: (I just had to upgrade VS Code to 1.54 first)
